pkg/servers/configs/configs.go

```diff
@@ -100,3 +115,18 @@ func loadConfigFromFile(configDir string) (*ControlplaneRunConfig, error) {
 
 	return c, nil
 }
+
+func (c *ControlplaneRunConfig) isClusterSupported() bool {
```

Comment:
we should not care about the cluster type, we should care about the service type, right?